workflow-dev/tpl/deis-nsqd-rc.yaml

```diff
@@ -0,0 +1,49 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifests/deis-nsqd-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-nsqd
+  namespace: deis
+  labels:
+    heritage: deis
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-nsqd
+  template:
+    metadata:
+      labels:
+        app: deis-nsqd
+    spec:
+      serviceAccount: deis-nsqd
+      containers:
+        - name: deis-nsqd
+          image: quay.io/{{.nsqd.org}}/nsqd:{{env "NSQ_GIT_TAG" | default .nsqd.dockerTag}}
+          imagePullPolicy: {{.nsqd.pullPolicy}}
+          command:
+          - /opt/nsq/bin/start-nsqd
```

Comment:
so i toyed with putting it in `/bin` but felt this was more in line with how we do our other images.